rpc/src/server.rs

```diff
@@ -22,6 +30,10 @@ use tokio_util::codec::{FramedRead, FramedWrite, LinesCodec, LinesCodecError};
 use tower_http::cors::CorsLayer;
 use tower_http::timeout::TimeoutLayer;
 
+const DEFAULT_BATCH_LIMIT: usize = 2000;
+
+static JSONRPC_BATCH_LIMIT: once_cell::sync::OnceCell<usize> = once_cell::sync::OnceCell::new();
```

Comment:
Why do we need a `JSONRPC_BATCH_LIMIT`? Can we only use `DEFAULT_BATCH_LIMIT` instead?

Reply:
removed `DEFAULT_BATCH_LIMIT` now, as discussed, we don't add default limit for user.